google-cloud-dlp/.yardopts

```diff
@@ -0,0 +1,8 @@
+--no-private
+--title=DLP API
+--exclude lib/google/cloud/dlp/v2beta1
```

Comment:
Yeah. that sounds right. I think I chose the wrong path in the template because I used an api where the proto path and the gapic path are the same as a guide. @geigerj we should figure out what to do for situations where the proto path and the gapic path are the same.